guides/common/modules/ref_pxe-loaders.adoc

```diff
@@ -0,0 +1,84 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="pxe-loaders"]
+= PXE loaders
+
+In {Project}, a PXE loader defines the `filename` DHCPv4 option to use during provisioning.
+{Project} deploys the required files to TFTP {SmartProxy}.
+
+.Limitations
+* {Project} can deploy the required booting files only for the `x86_64` architecture.
```

Comment:
Any source for this? With Grub 2 you can clearly see it includes the architecture in the filename and there was code written explicitly to support non-x86_64.